packages/opam-build/opam-build.0.2.5/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "An opam plugin to build projects"
+maintainer: "Kate <kit-ty-kate@outlook.com>"
```

Comment:
```suggestion
maintainer: "Kate <kit-ty-kate@outlook.com>"
description: """
opam-build allows to build any project easily with just one command: opam build. This will setup a local switch and install all the required dependencies.
"""
```